source/aws_imds_client.c

```diff
@@ -28,7 +28,7 @@
 #define IMDS_RESPONSE_TOKEN_SIZE_INITIAL 64
 #define IMDS_RESPONSE_SIZE_LIMIT 65535
 #define IMDS_CONNECT_TIMEOUT_DEFAULT_IN_SECONDS 2
-#define IMDS_DEFAULT_RETRIES 1
```

Comment:
What is the default chain timeout now when it is not running on EC2 without any credentials?

Reply:
reverted the change, was trying to follow other SDKs